aws/structure.go

```diff
@@ -955,10 +955,25 @@ func flattenNeptuneParameters(list []*neptune.Parameter) []map[string]interface{
 }
 
 // Flattens an array of Parameters into a []map[string]interface{}
-func flattenDocDBParameters(list []*docdb.Parameter) []map[string]interface{} {
+func flattenDocDBParameters(list []*docdb.Parameter, d *schema.ResourceData) []map[string]interface{} {
 	result := make([]map[string]interface{}, 0, len(list))
 	for _, i := range list {
 		if i.ParameterValue != nil {
+			name := aws.StringValue(i.ParameterName)
+
+			// Check if any non-user parameters are specified in the configuration.
+			parameterFound := false
+			for _, configParameter := range d.Get("parameter").(*schema.Set).List() {
```

Comment:
This may only work on non-imported resources. The testing should include an `ImportStateVerify` (and `ImportStateVerifyIgnore` if necessary) step to prove or disprove this functionality. If it does not work on imported resources, it may be worth noting in the documentation as well with something along the lines of:

```markdown
Setting parameters to system default values may show a difference on imported resources.
```